### PR TITLE
[11.0][ADD] web_company_title

### PR DESCRIPTION
**Pull request by Tardo**
This module show the company name in the browser title-bar.

### CODE REVIEW

File: web_company_systray/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Odoo, Open Source Web Company Systray
+# Copyright (C) 2019 Alexandre Díaz <dev@redneboa.es>
+#
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).#
+{
+    'name': "Web Company Systray",
```
Comment (nikul-serpentcs) [inline web_company_systray/__manifest__.py]: Add `website` key

**Comment by simahawk** [comment]
can't we simply add current company name in the top bar?

**Comment by tbaden** [comment]
is there really a use case? in multi company you can simply see it and in single company there is only one company. normally the user should know the name of his company

**Comment by Tardo** [comment]
@tbaden Is a 'visual helper' when you have to manage a lot of instances. 
The original idea is taken from @OsoTranquilo (used in real world)

@simahawk You mean not use menu-bar and use title-bar? Ummm if is this.. yeah, sounds good for me! thx again ;)

**Review by tarteo — APPROVED** [review]

**Review by nikul-serpentcs — APPROVED** [review]
code review LGTM :+1:

**Comment by Vassy** [comment]
Company name displays only on english, not in users language, e.g. bulgarian.

**Pull request merged**